cni/azure-windows.conflist

```diff
@@ -40,6 +40,46 @@
                         "DestinationPrefix": "10.0.0.0/8",
                         "NeedEncap": true
                     }
+                },
+                {
+                    "Name":  "EndpointPolicy",
+                    "Value":  {
+                        "Type":  "ACL",
+                        "Action":  "Allow",
+                        "Direction":  "Out",
+                        "RemoteAddresses":  "168.63.129.16/32",
+                        "Protocols": "17",
+                        "RemotePorts":  "53",
+                        "Priority":  200
+                    }
+                },
+                {
+                    "Name":  "EndpointPolicy",
```

Comment:
Remove 2 spaces after : 
We have one space for all the other settings

Reply:
done.